xla/service/gpu/fusions/triton/triton_fusion_emitter_device_legacy_test.cc

```diff
@@ -162,26 +162,6 @@ TEST_F(TritonGemmTest, RejectDotInt4HLO) {
               StatusIs(tsl::error::INVALID_ARGUMENT));
 }
 
-TEST_F(TritonGemmTest, RejectInt4NegatePlusConvertHLO) {
```

Comment:
What's the expectation here, that it's fused?

If so, should we check correctness of the lowering?

Reply:
Restored the tests, removed "Reject" from the test name.